src/acquisition/dotnetAcquisition.ts

```diff
@@ -76,7 +76,7 @@ export async function dotnetAcquire(
         }
         return await acquisitionWorker.acquire(version, telemetryProperties);
     } catch (err) {
-        const linkMessage = `>>>> This extension requires .NET Core for full functionality, but we were unable to download and install a local copy for the extension. If this error persists, please see https://aka.ms/vscode-armtools-dotnet for troubleshooting tips.`;
+        const linkMessage = `>>>> This extension requires .NET Core for full functionality, but we were unable to download and install a local copy for the extension. If this error persists, please see https://aka.ms/vscode-armtools-dotnet for troubleshooting tips. `;
```

Comment:
> ` [](start = 265, length = 2)

Intentional?

Reply:
Yeah, in the errors I was seeing no space after this before the next error.  I'm reworking this a little in another PR, so not worrying about improving it now.